pytools/__init__.py

```diff
@@ -411,9 +411,14 @@ class RecordWithoutPickling:
     """
 
     __slots__: ClassVar[List[str]] = []
-    fields: ClassVar[Set[str]]
+
+    # A dict, not a set, to maintain a deterministic iteration order
+    fields: ClassVar[Dict[str, None]]
 
     def __init__(self, valuedict=None, exclude=None, **kwargs):
+        from warnings import warn
+        warn("pytools.Record is deprecated. Use dataclasses instead.")
```

Comment:
Also remove from docs.

Reply:
Is it documented somewhere? I could not find it.

Reply:
Looks like we already got rid of it. Good! Nevermind then. :slightly_smiling_face: